### PR TITLE
CDAP-13151 - Fixing bug with JSON depth parsing. 

### PR DESCRIPTION
**Pull request by nitinmotgi**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -79,7 +79,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>4.3.4</cdap.version>
```
Comment (nitinmotgi) [inline pom.xml]: Need to fix this to use non snapshot version.

File: pom.xml [hunk 2]
```diff
@@ -129,6 +129,10 @@
       <id>sonatype-snapshots</id>
       <url>https://oss.sonatype.org/content/repositories/snapshots</url>
     </repository>
+    <repository>
```
Comment (sreevatsanraman) [inline pom.xml]: you don't need this. Since we already have the snapshot repo in the pom.xml

**Comment by sreevatsanraman** [comment]
one minor comment. LGTM otherwise.

**Comment by nitinmotgi** [comment]
@sreevatsanraman addressed comments.

**Comment by sreevatsanraman** [comment]
LGTM

**Pull request merged**